pepper-apis/dsm-core/src/main/resources/liquibase/singular/DDP-8810_feild_settings_details_column.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="DDP-8810_field_settings_details_columns" author="pegah">
+        <addColumn tableName="field_settings">
+            <column name="details" type="VARCHAR(500)" afterColumn="actions"></column>
```

Comment:
what is `details` for?

Reply:
So the ticket asks for a way to dynamically change some text area sizes and limits, we will add the new "details" of each field in this `details` column. ( So for the fields added per DDP-8808 it will be like `{size:100}`. we can use this column for other features of the fields in future too ( color, font,  readonly, etc)